engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/SubWorkflowExecutionActor.scala

```diff
@@ -142,6 +142,7 @@ class SubWorkflowExecutionActor(key: SubWorkflowKey,
         case Some(id) =>
           pushWorkflowEnd(id)
           pushExecutionEventsToMetadataService(key, eventList)
+          context.system.eventStream.publish(SubWorkflowEnd(id))
```

Comment:
This seems to be the happy path but if for some reason this actor blows up we might not end up here which would forever mess up the sub workflow count. Maybe the WMA could watch the SWEA when it receives the Start event, and decrement then ?
If you do end up going the unit test route, killing this actor on purpose would be something worth testing too I think